examples/example_i2c_master.c

```diff
@@ -0,0 +1,99 @@
+#include <esp_err.h>
```

Comment:
Can you please take the time have the original example in a file like this? Not really required for me to merge the PR but I would appreciate it.

Reply:
Will do